bombastic/walker/walker.sh

```diff
@@ -18,27 +27,24 @@ help()
    # Display Help
    echo "Get Sboms from prodsec and push to bombastic."
    echo
-   echo "sbomwalker [OPTIONS] BOMBASTIC_API_URL"
+   echo "sbom_uploader [-h] SBOM_URL BOMBASTIC_API_URL"
    echo "options:"
-   echo "   -s <URL>  Specify the source URL. Defaults to https://access.redhat.com/security/data/sbom/beta/index.txt"
    echo "   -h        Print this Help."
    echo
    echo "example:"
-   echo "sbomwalker localhost:8080"
-   echo "sbomwalker -s https://access.redhat.com/security/data/sbom/beta/index.txt localhost:8080"
+   echo "sbomwalker https://access.redhat.com/security/data/sbom/beta/sbdx/3amp-2.json.bz2 localhost:8080"
    echo
```

Comment:
Which one is it, sbom_uploader or sbomwalker? :)

Reply:
Fixed with `$0`  :)